src/authz-module/libraries-manager/ErrorPage/index.tsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from 'react';
+import { FallbackProps } from 'react-error-boundary';
+import { getConfig } from '@edx/frontend-platform';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import {
+  Button, Container, Hyperlink, Row,
+} from '@openedx/paragon';
+import { CustomErrors, ERROR_STATUS } from '@src/constants';
+
+import messages from './messages';
+
+const getErrorConfig = ({ errorMessage, errorStatus }) => {
+  if (errorMessage === CustomErrors.NO_ACCESS || ERROR_STATUS.NO_ACCESS.includes(errorStatus)) {
+    return ({
+      title: messages['error.page.title.noAccess'],
+      description: messages['error.page.message.noAccess'],
+      statusCode: errorStatus || ERROR_STATUS.NO_ACCESS[0],
+      showBackButton: true,
+    });
+  }
+  if (errorMessage === CustomErrors.NOT_FOUND || ERROR_STATUS.NOT_FOUND.includes(errorStatus)) {
+    return ({
+      title: messages['error.page.title.notFound'],
+      description: messages['error.page.message.notFound'],
+      statusCode: errorStatus || ERROR_STATUS.NOT_FOUND[0],
+      showBackButton: true,
+    });
+  }
+  if (errorMessage === CustomErrors.SERVER_ERROR || ERROR_STATUS.SERVER_ERROR.includes(errorStatus)) {
+    return ({
+      title: messages['error.page.title.server'],
+      description: messages['error.page.message.server'],
+      statusCode: errorStatus || ERROR_STATUS.SERVER_ERROR[0],
+      showBackButton: true,
+      showReloadButton: true,
+    });
+  }
+  return ({
+    title: messages['error.page.title.generic'],
+    description: messages['error.page.message.generic'],
+    showBackButton: true,
+    showReloadButton: true,
+  });
+};
+
+const librariesUrl = () => `${getConfig().AUTHORING_BASE_URL}/authoring/libraries`;
```

Comment:
I am not getting why this one is a function and not just a constant.
```suggestion
const LIBRARIES_URL = `${getConfig().AUTHORING_BASE_URL}/authoring/libraries`;
```
And maybe it can be moved to the constants file.